src/Android/Avalonia.Android/Platform/SkiaPlatform/TopLevelImpl.cs

```diff
@@ -624,6 +624,12 @@ public override bool PerformEditorAction([GeneratedEnum] ImeAction actionCode)
                         _inputMethod.IMM.HideSoftInputFromWindow(_inputMethod.View.WindowToken, HideSoftInputFlags.ImplicitOnly);
                         break;
                     }
+                case ImeAction.Next:
+                    {
+                        var focusedElement = FocusManager.GetFocusManager(_toplevel.InputRoot).GetFocusedElement();
```

Comment:
Can we share this logic somewhere so every platform can call the same logic?

`InputMethod.ExecuteAction(ActionEnum)`

Not sure where this logic should be implemented

Reply:
Ideally, someone could register some delegate that gets executed for a given action type